website/cue/reference/components/sources/base/heroku_logs.cue

```diff
@@ -439,12 +439,16 @@ base: components: sources: heroku_logs: configuration: {
 		description: """
 			A list of URL query parameters to include in the log event.
 
+			Accepts the wildcard (`*`) character for query parameters matching a specified pattern.
+
+			Specifying "*" results in all query parameters included in the log event.
```

Comment:
I'm not 100% sure what this means - can you elaborate?

Reply:
Given the following query parameters: `hello=world&source=test&application=demo`:
##### Exact
```yaml
  sources:
    exact:
      type: http_server
      headers:
        - "*"
      query_parameters:
        - "hello"
```
Will emit `"message": {"query_parameters": {"hello": "world"}}` as part of the event

##### Wildcard
```yaml
  sources:
    exact:
      type: http_server
      headers:
        - "*"
      query_parameters:
        - "*"
```
Will emit `"message": {"query_parameters": {"hello": "world", "source": "test", "application": "demo"}}` as part of the event

##### Prefix
```yaml
  sources:
    exact:
      type: http_server
      headers:
        - "*"
      query_parameters:
        - "s*"
```
Will emit `"message": {"query_parameters": {"source": "test"}}` as part of the event